make/source.mk

```diff
@@ -117,6 +117,7 @@ FC_SRC = \
             rx/nrf24_h8_3d.c \
             rx/nrf24_syma.c \
             rx/nrf24_v202.c \
+            rx/nrf24_kn.c \
```

Comment:
Considering that this will only ever be used by 'speciality boards', I think it would make sense to remove this from here, and instead add it to the `target.mk` for these boards. This will eliminate touching this file for the 140+ boards we build (even if the conditionals mean the contents aren't compiled, just touching it and evaluating the contents takes time, times 140+).

@martinbudden: What's your take on this? (Also, this could probably be done for the other `nrf24_` protocols as well.)

Reply:
Well, I'd like to measure how much time it actually saves before we do this. But if there is a measured significant saving, I am OK with it.

Reply:
@martinbudden:

Timing call:
`make clean_all; time make targets-group-1`

With nrf24 drivers:

```
real    5m11.562s
user    6m6.903s
sys 0m40.928s
```

Change:

```
-            drivers/rx/rx_nrf24l01.c \
-            rx/nrf24_cx10.c \
-            rx/nrf24_inav.c \
-            rx/nrf24_h8_3d.c \
-            rx/nrf24_syma.c \
-            rx/nrf24_v202.c \
```

Without nrf24 drivers:

```
real    4m43.633s
user    5m47.378s
sys 0m36.517s
```